platform/jobs/exampleJob.groovy

```diff
@@ -117,7 +114,7 @@ secretMap.each { jobConfigs ->
        }
 
        /* Post Build Steps */
-       /* Archive artifacts, archive jUnit Reports, send an email, message on hipchat */
+       /* Archive artifacts, archive jUnit Reports, send an email */
```

Comment:
Should the example job be updated to also message Slack, since it was doing HipChat before?